libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1822,39 +1821,6 @@ void GCS_MAVLINK::send_scaled_pressure3()
     send_scaled_pressure_instance(2, mavlink_msg_scaled_pressure3_send);
 }
 
-void GCS_MAVLINK::send_sensor_offsets()
-{
-    const AP_InertialSensor &ins = AP::ins();
-    const Compass &compass = AP::compass();
-
-    // run this message at a much lower rate - otherwise it
-    // pointlessly wastes quite a lot of bandwidth
-    if (send_sensor_offsets_counter++ < 10) {
```

Comment:
Ha! This defeats a lot of the scheduling/buffering code as well because it's harder to land on the right order with stuff like this confusing the buffer backoffs :P